plugins/backend/catalog-backend-module-okta/src/providers/OktaUserProvider.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2021 Larder Software Limited
```

Comment:
I think in the notice the date should be the current one, in this case the year 2022 when this was created.

```suggestion
 * Copyright 2022 Larder Software Limited
```

Reply:
done